pytorch_lightning/accelerators/accelerator.py

```diff
@@ -35,7 +35,13 @@ def setup_environment(self, root_device: torch.device) -> None:
 
         This is called before the LightningModule/DataModule setup hook which allows the user to access the accelerator
         environment before setup is complete.
+
+        Raises:
+            RuntimeError:
+                If corresponding hardware is not found.
         """
+        if not self.is_available():
+            raise RuntimeError(f"{self.__class__.__qualname__} is not configured to run on this hardware.")
```

Comment:
If I dont' have a GPU, this says "GPUAccelerator is not configured to run on this hardware"

This makes it sounds like there's a setting for GPUAccelerator that would make it work, when in reality I don't even have the hardware.

I would suggest rewording to "GPUAccelerator cannot run on this hardware" or something similar